spec/validators/representation_order_validator_spec.rb

```diff
@@ -176,6 +176,45 @@
         end
       end
     end
+
+    context 'when future dates are allowed' do
+      let(:reporder) { build(:representation_order, defendant:, representation_order_date: rep_order_date) }
+
+      before do
+        allow(ENV).to receive(:fetch).with('ALLOW_FUTURE_DATES', nil).and_return('true')
+        allow(ENV).to receive(:fetch).with('ENV', nil).and_return('dev')
```

Comment:
To ensure that other environment variables are not affected;

```suggestion
        allow(ENV).to receive(:fetch).and_call_original
        allow(ENV).to receive(:fetch).with('ALLOW_FUTURE_DATES', nil).and_return('true')
        allow(ENV).to receive(:fetch).with('ENV', nil).and_return('dev')
```

Reply:
👍